### DISCUSSION TITLE
What is the difference compared to `aws sso login`?

### DISCUSSION THREAD

**Discussion by scmrus** [theurichde/go-aws-sso#183] [Q&A] [answered]
I'm sorry, this might be a silly question, but what is the difference between `aws sso login` and `go-aws-sso`?

**Comment by theurichde** [comment, marked as answer]
Hi there.

The ["But Why...?"](https://github.com/theurichde/go-aws-sso?tab=readme-ov-file#but-why-) section explains it.

Try running `aws sso login` without Python installed - you will fail.
Try choosing your desired account and role directly from 30 accounts with 4 roles each - you can't.

`go-aws-sso` is intended to make your life easier on command line. But as always, your experience and taste may differ.

**Reply by scmrus** [reply]
So, it's essentially for someone who doesn't want to install the AWS CLI or configure the ~/.aws/config. Thank you for the clarification!

I'm curious because I have an AWS-related tool written in Go (ssh over ssm) and was wondering if I could take some ideas to avoid the AWS CLI dependency using your tool.
